Dockerfile

```diff
@@ -3,37 +3,33 @@ FROM ubuntu:20.04
 RUN groupadd -g 5000 sync-engine \
   && useradd -d /home/sync-engine -m -u 5000 -g 5000 sync-engine
 
-ENV TZ="Etc/GMT"
-ENV DEBIAN_FRONTEND=noninteractive
+ENV \
+  TZ="Etc/GMT" \
+  LANG="en_US.UTF-8" \
+  LC_ALL="en_US.UTF-8" \
+  DEBIAN_FRONTEND=noninteractive \
+  PATH="/opt/venv/bin:$PATH"
+
 ARG BUILD_WEEK=0
 RUN echo $BUILD_WEEK && apt-get update \
   && apt-get dist-upgrade -y \
-  && apt-get install -y \
+  && apt-get install --no-install-recommends -y \
     tzdata \
-    build-essential \
+    locales \
+    make \
     curl \
-    dnsutils \
+    gpg \
+    gpg-agent \
+    dirmngr \
```

Comment:
Those were previously installed from recommended, we need those for some things devops uses in the upstream image.